src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx

```diff
@@ -84,6 +85,9 @@ class RaceTrackerMap extends React.Component<IRaceTrackerMapProps, any> {
               ]
             },
             properties: {
+              description: `<h3>Racer ID: `+trackLeader['trackleaders_racer_ID'][0]+`</h3> 
```

Comment:
If you're going to use the backtick ( ` ) to do template strings, I would recommend that you make use of the string interpolation. This block of code becomes
```
properties: {
  description: `
    <h3>
      Racer ID: ${trackLeader['trackleaders_racer_ID'][0]}
    </h3> 
    <strong>
      Device Battery State: ${trackLeader.message[0]['batteryState'][0]}
    </strong>
  `,
  speed: 10
}
```

This way, it's a lot more clear what the structure is of what you're trying to do, and it's more readable.

Reply:
The issue with your popup is the fact that you're passing a string containing HTML. `{popup.description}` only sees a string, not the fact that it needs to render HTML. When you click on a marker on the map, you need to `setState` the info you need. And inside of `<Popup></Popup>` you need to write out the HTML structure with JSX that you are currently putting in `description`.